docs/delegate.md

```diff
@@ -0,0 +1,97 @@
+# Delegation
+
+For a more flexible configuration, you can have `server_name`
```

Comment:
more flexible than what? we'll have to rephrase this now that it's in a different doc.